gulpfile.js

```diff
@@ -1,5 +1,7 @@
 const gulp = require('gulp');
+const shell = require('gulp-shell');
 const clangFormat = require('gulp-clang-format');
+const os = require('os');
```

Comment:
Let's move `const os` to the top since it's a Node core module.

Reply:
yep. updated.